docs/semantics.md

```diff
@@ -228,6 +228,8 @@ However, implicit directories does have drawbacks:
 
 Alternatively, users can create a script which lists the buckets and creates the appropriate objects for the directories so that the ```--implicit-dirs``` flag is not used. 
```

Comment:
This line is a duplicate of what I've added in the note right after it. Let's merge this into the Node to avoid redundancy.